src/test/regress/sql/multi_truncate.sql

```diff
@@ -221,3 +221,75 @@ DELETE FROM pg_dist_partition WHERE logicalrelid = 'test_local_truncate'::regcla
 SELECT * FROM test_local_truncate;
 
 DROP TABLE test_local_truncate;
+
+CREATE TABLE test_seq_truncate (a int);
+INSERT INTO test_seq_truncate SELECT i FROM generate_series(0, 100) i;
+
+SELECT create_distributed_table('test_seq_truncate', 'a');
+
+-- disable the 2PC recovery for now
+ALTER SYSTEM SET citus.recover_2pc_interval TO -1;
+SELECT pg_reload_conf();
```

Comment:
we should probably add a note somewhere that this test file cannot be used in parallel with others.

Reply:
I'm planning to move these tests to the new file added in #2189. I'll rename `sequential_ddl.sql` to `sequential_modification.sql`.